app/views/schools/audits/index.html.erb

```diff
@@ -1,24 +1,23 @@
-<% content_for :page_title, "#{@school.name} Audits" %>
+<% content_for :page_title, "#{@school.name} " + t('schools.audits.index.title') %>
 
 <div class="row">
   <div class="col-md-12">
-    <h1>Energy audits</h1>
+    <h1><%= t('schools.audits.index.energy_audits') %></h1>
   </div>
 </div>
 
 <% if @audits.any? %>
   <div class="row">
     <div class="col-md-12">
-      <p>The Energy Sparks team have carried out the following audits at this school.
-      </p>
+      <p><%= t('schools.audits.index.message') %>.</p>
 
       <table class="table" id="audits">
         <thead>
           <tr>
-            <th>Name</th>
-            <th>Completed on</th>
+            <th><%= t('schools.audits.index.name') %></th>
+            <th><%= t('schools.audits.index.completed_on') %></th>
             <% if can? :manage, @activity %>
-              <th>Actions</th>
+              <th><%= t('schools.audits.index.actions') %></th>
```

Comment:
and this? common.labels.actions ? Not sure if it's there but feels like it should be

Reply:
:+1: